tests/baselines/reference/api/typescript.d.ts

```diff
@@ -3071,6 +3071,7 @@ declare namespace ts {
         scanJsxIdentifier(): SyntaxKind;
         scanJsxAttributeValue(): SyntaxKind;
         reScanJsxToken(): JsxTokenSyntaxKind;
+        reScanLessThanToken(): SyntaxKind;
```

Comment:
Disgustingly, the corresponding method for a greater-than is called `reScanGreaterToken`. Not sure if we should be consistent here.